conda/shell/etc/profile.d/conda.sh

```diff
@@ -29,7 +29,7 @@ __conda_activate() {
 
 __conda_reactivate() {
     # FUTURE: conda 25.9, remove this function
-    echo `__conda_reactivate` is deprecated and will be removed in 25.9. Use `__conda_activate reactivate` instead. 1>&2
+    echo "'__conda_reactivate' is deprecated and will be removed in 25.9. Use '__conda_activate reactivate' instead." 1>&2
```

Comment:
Thank you, Jaime. :pray:

I have one UX question based on https://github.com/conda-forge/miniforge/issues/700#issuecomment-2568526971.

If users of conda or mamba have their setup uses `__conda_reactivate`, will they know:
 - 1) why this warning is show
 - 2) where the warning is issued from
 - 3) how to resolve the problem (here replace instance of `__conda_reactivate` with `__conda_activate reactivate)

?

Reply:
AFAICS this is just a standard error message, no fancy deprecation framework in use, etc. So if that information needs to be included, we need to add it manually. Something like this?

```suggestion
    echo "Warning(conda.sh): '__conda_reactivate' is deprecated and will be removed in 25.9. '__conda_activate reactivate' is the recommended replacement. Consider updating conda and/or mamba to make this warning disappear." 1>&2
```

I don't think we have a way of detecting how the shell function was called without introducing a bigger surface for potential problems.

Reply:
LGTM!